requirements-base.txt

```diff
@@ -55,7 +55,8 @@ requests>=2.25.1
 rfc3339-validator>=0.1.2
 rfc3986-validator>=0.1.1
 # [end] jsonschema format validators
-sentry-arroyo>=2.7.1
+sentry-arroyo[json]>=2.7.1
+sentry-kafka-schemas==0.0.11
```

Comment:
Should probably be `>=` like all the rest of the Sentry dependencies. Please update to 0.0.16 also

Reply:
Are there steps I need to take or is the new workflow automated?

Reply:
you should be good